graphics/lvgl/lv_conf/Kconfig

```diff
@@ -0,0 +1,1110 @@
+# Kconfig file for LVGL v8.0
+
+menu "LVGL configuration"
```

Comment:
Kconfig files in NuttX are indented with tabs.

Reply:
This Kconfig is copied from lvgl: https://github.com/lvgl/lvgl/blob/release/v8.3/Kconfig.
Prevent failure when executing `tools/configure.sh`. If the user needs to modify the lvgl version, keeping the original format is conducive to file comparison.

Reply:
I am not sure if this is a valid argument.

As an example, most of the Kconfig options from the ESP32 family of chips are similar to ESP-IDF's, which also use whitespaces for indentation instead of tabs. Yet, the Kconfig files follow the NuttX style.

Reply:
Taking it to an extreme scenario, suppose that for every third-party project to be included in `apps` that also uses Kconfig, each one follows a different formatting style.
Can you imagine the mess that the nuttx-apps repository would become?

So I am in favor of creating no exceptions to this.
Copying the options is already a huge benefit. Adapting to NuttX formatting is a small price to pay for consistency.

Reply:
@FASTSHIFT let's change space to tab and upstream to lvgl.